packages/dapp-svelte-wallet/api/test/test-lib-wallet.js

```diff
@@ -262,22 +268,23 @@ test('lib-wallet dapp suggests issuer, instance, installation petnames', async t
     // invite purse balance is rendered with petnames. We should see
     // unnamed first, then the petnames after those are suggested by
     // the dapp.
-    const { publicAPI } = await E(zoe).getInstanceRecord(instanceHandle);
-    const invite2 = await E(publicAPI).makeInvite();
+    /** @type {{ makeInvitation(): Invitation }} */
+    const publicAPI = await E(zoe).getPublicFacet(instance);
+    const invite2 = await E(publicAPI).makeInvitation();
     const {
       value: [{ handle: inviteHandle2 }],
     } = await E(inviteIssuer).getAmountOf(invite2);
-    await wallet.deposit('Default Zoe invite purse', invite2);
+    wallet.deposit('Default Zoe invite purse', invite2);
```

Comment:
```suggestion
    await wallet.deposit('Default Zoe invite purse', invite2);
```

Reply:
Turns out wallet.deposit should be an async method.  Will address in a coming PR.